### PR TITLE
#69 reworked the doc with previous/next links

### PR DESCRIPTION
**Pull request by etorreborre**
added another maintainer
added a SECURITY file

The doc should now read with a better flow from installation to testing.

Review @jcranky

### CODE REVIEW

File: doc/boilerplate.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+### Remove boilerplate with shapeless
```
Comment (jcranky) [inline doc/boilerplate.md]: This file is not linked in `README.md`, should it?

Comment (etorreborre) [inline doc/boilerplate.md]: Yes it should. I'll do that

Comment (etorreborre) [inline doc/boilerplate.md]: Done in fixup.

**Comment by jcranky** [comment]
Since when we have codacy here? I like it :)

**Comment by jcranky** [comment]
(but it seems to be running under your name there, not Zalando)

**Comment by jcranky** [comment]
:+1:

**Comment by etorreborre** [comment]
👍

**Pull request merged**